Tests/WebSocketKitTests/SSLTestHelpers.swift

```diff
@@ -0,0 +1,142 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Can you add a NOTICE.txt as well if we're taking code from NIO. E.g. https://github.com/vapor/jwt-kit/blob/main/NOTICES.txt